subworkflows/local/decontamination.nf

```diff
@@ -22,7 +22,7 @@ workflow DECONTAMINATION {
 
     ALIGNMENT( to_align, false )
 
-    SAMTOOLS_BAM2FQ( ALIGNMENT.out.bam.map { meta, ref_fasta, bam, bai -> [ meta, bam ] }, true )
+    SAMTOOLS_BAM2FQ( ALIGNMENT.out.bam.map { meta, ref_fasta, bam, bai -> [ meta, bam ] }, reads.map { meta, reads -> meta.single_end == false } )
```

Comment:
To match the three elements tuple passed to sam2fastq, we need to change slightly this line:

`SAMTOOLS_BAM2FQ(  ALIGNMENT.out.bam.map  { meta, ref_fasta, bam, bai -> [ meta, bam, meta.single_end == false ] } )`